modules/500-upmeter/images/upmeter/werf.inc.yaml

```diff
@@ -0,0 +1,51 @@
+---
+artifact: {{ .ModuleName }}/build-upmeter-artifact
+from: {{ .Images.BASE_GOLANG_16_ALPINE }}
```

Comment:
please use higher version of golang, to prevent CVE's appearing

Reply:
done